course/src/isometry.cpp

```diff
@@ -2,16 +2,234 @@
  * Isometry library
  * Author: Agustin Alba Chicar, 2019
  * Author: Gerardo Puga, 2020
+ * Copyright 2020 Ekumen"
  */
 
 #include <isometry/isometry.hpp>
 
 namespace ekumen {
 namespace math {
 
-/*
- * your code here
- */
+Vector3::Vector3(double x, double y, double z) {
+  this->x_ = x;
+  this->y_ = y;
+  this->z_ = z;
+}
+
+Vector3::Vector3(std::initializer_list<double> list) {
+  if (list.size() != 3) {
+    throw std::out_of_range("Incorret size of list");
+  }
+  auto it = list.begin();
+  this->x_ = *it;
+  it++;
```

Comment:
Please make a habit of using pre-increment instead of post-increment. 

Post increment may do an additional copy that gets thrown away.